save-frontend/src/main/kotlin/org/cqfn/save/frontend/App.kt

```diff
@@ -142,6 +145,39 @@ class App : ComponentWithScope<PropsWithChildren, AppState>() {
                                 }
                             }
 
+                            Route {
+                                attrs {
+                                    path = "/:user/Settings/Profile"
```

Comment:
```suggestion
                                    path = "/:user/settings/profile"
```
Why capital letters? Usually, everything is lowercase in URLs

Reply:
Fixed